rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -82,7 +87,29 @@ void SqliteStorage::open(
     initialize();
   }
 
-  ROSBAG2_STORAGE_DEFAULT_PLUGINS_LOG_INFO_STREAM("Opened database '" << uri << "'.");
+  uri_ = uri;
+  io_flag_ = io_flag;
+  max_bagfile_size_ = max_bagfile_size;
+
+  ROSBAG2_STORAGE_DEFAULT_PLUGINS_LOG_INFO_STREAM("Opened database '" << database_name_ << "'.");
+}
+
+uint64_t SqliteStorage::get_db_size()
+{
+  struct stat stat_res;
```

Comment:
In C, `struct` is mandatory, but not in C++.

```suggestion
 stat stat_res;
```